eZ/Publish/Core/Persistence/Legacy/Content/Section/Gateway/ExceptionConversion.php

```diff
@@ -145,6 +145,42 @@ public function countContentObjectsInSection($id)
     }
 
     /**
+     * Counts the number of role policies using section with $id in their limitations.
+     *
+     * @param int $id
+     *
+     * @return int
```

Comment:
We don't do that in any of the other `ExceptionConversion` classes either afaict, so I'd deal with that in a separate issue if we're doing it. Don't know why it has been skipped.